widget/list.go

```diff
@@ -186,6 +187,11 @@ func (l *List) scrollTo(id ListItemID) {
 		l.scroller.Offset.Y = y + lastItemHeight - l.scroller.Size().Height
 	}
 	l.offsetUpdated(l.scroller.Offset)
+	defer func() {
+		if f := l.OnFocus; f != nil {
+			f(id)
+		}
+	}()
```

Comment:
Also, please add some unit tests to verify the behavior, see the existing examples in list_test.go

Reply:
Done!
Thanks a lot for your hints, let me know if it can be enough